api/examples/flow-get-200-multi.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "e85efab4-993b-4ad6-9af3-4cd8d0d38860",
+    "source_id": "a77d0061-0878-4e8a-a114-772d03f952c1",
+    "format": "urn:x-nmos:format:data",
+    "generation": 0,
+    "created": "2023-12-14T16:34:58.131620",
+    "tags": {
+        "ingested_by": "ingest_service_api"
+    },
+    "codec": "application/ttml+xml",
+    "container": "application/ttml+xml",
```

Comment:
`video/mp2t` would work better in this example. However, this is the container in the TAMS and as this is a multi-essence flow there may not be a container if there is no media object, i.e. mono-essence is stored. The `container` property would need to be optional as well.